src/style.css

```diff
@@ -1,3 +1,70 @@
-body {
+.ttq {
+  margin: 0;
+}
```

Comment:
You should avoid using cryptic class names like '.ttq'. It's not clear what this class is for. Try to use more descriptive class names that give an idea about the element's purpose or the styles that the class applies.